xray/services/scan.go

```diff
@@ -111,6 +112,10 @@ func (ss *ScanService) ScanGraph(scanParams XrayGraphScanParams) (string, error)
 			return "", fmt.Errorf("failed sending Git Info to XSC service, error: %s ", err.Error())
 		}
 		scanParams.MultiScanId = multiScanId
+		err = os.Setenv("JF_MSI", multiScanId)
+		if err != nil {
+			return "", fmt.Errorf("failed setting msi as environment variable")
+		}
```

Comment:
The above code can be simplified as follows. In addition, the error created should be wrapped with `errorutils.CheckErrorf(err)` iin this case, beucase `os.Setenv`, which may return the error, is an external API to the `jfrog-client-go` codebase. Also notice that in the above code the original error is lost, and the below code fixes that as well.
```
if err = os.Setenv("JF_MSI", multiScanId); err != nil {
	errorutils.CheckErrorf("failed setting msi as environment variable. Cause: %s", err.Error())
}
```